{"level":"error","message":"Error during Data Source:  connect ECONNREFUSED ::1:27017","reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Unknown"},"stack":"MongoServerSelectionError: connect ECONNREFUSED ::1:27017\n    at Timeout._onTimeout (D:\\projects\\recipes\\server\\node_modules\\mongodb\\src\\sdam\\topology.ts:567:30)\n    at listOnTimeout (node:internal/timers:573:17)\n    at processTimers (node:internal/timers:514:7)","timestamp":"2025-03-17T17:24:58.584Z"}
{"level":"error","message":"Error during Data Source:  getaddrinfo ENOTFOUND cluster0.elojd0q.mongodb.net","reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Unknown"},"stack":"MongoServerSelectionError: getaddrinfo ENOTFOUND cluster0.elojd0q.mongodb.net\n    at Timeout._onTimeout (D:\\projects\\recipes\\server\\node_modules\\mongodb\\src\\sdam\\topology.ts:567:30)\n    at listOnTimeout (node:internal/timers:573:17)\n    at processTimers (node:internal/timers:514:7)","timestamp":"2025-03-17T17:25:55.202Z"}
{"level":"error","message":"Error register user:  User with email exists","stack":"Error: User with email exists\n    at D:\\projects\\recipes\\server\\src\\services\\auth.service.ts:15:9\n    at Generator.next (<anonymous>)\n    at fulfilled (D:\\projects\\recipes\\server\\src\\services\\auth.service.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-17T17:29:31.446Z"}
{"level":"error","message":"SyntaxError: Expected double-quoted property name in JSON at position 58\n    at JSON.parse (<anonymous>)\n    at parse (D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\types\\json.js:92:19)\n    at D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\read.js:128:18\n    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)\n    at invokeCallback (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:238:16)\n    at done (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:227:7)\n    at IncomingMessage.onEnd (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:287:7)\n    at IncomingMessage.emit (node:events:513:28)\n    at endReadableNT (node:internal/streams/readable:1359:12)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)","timestamp":"2025-03-17T17:32:39.761Z"}
{"level":"error","message":"SyntaxError: Expected double-quoted property name in JSON at position 58\n    at JSON.parse (<anonymous>)\n    at parse (D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\types\\json.js:92:19)\n    at D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\read.js:128:18\n    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)\n    at invokeCallback (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:238:16)\n    at done (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:227:7)\n    at IncomingMessage.onEnd (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:287:7)\n    at IncomingMessage.emit (node:events:513:28)\n    at endReadableNT (node:internal/streams/readable:1359:12)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)","timestamp":"2025-03-17T17:32:52.988Z"}
{"level":"error","message":"SyntaxError: Expected double-quoted property name in JSON at position 58\n    at JSON.parse (<anonymous>)\n    at parse (D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\types\\json.js:92:19)\n    at D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\read.js:128:18\n    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)\n    at invokeCallback (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:238:16)\n    at done (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:227:7)\n    at IncomingMessage.onEnd (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:287:7)\n    at IncomingMessage.emit (node:events:513:28)\n    at endReadableNT (node:internal/streams/readable:1359:12)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)","timestamp":"2025-03-17T17:32:53.887Z"}
{"level":"error","message":"SyntaxError: Expected double-quoted property name in JSON at position 58\n    at JSON.parse (<anonymous>)\n    at parse (D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\types\\json.js:92:19)\n    at D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\read.js:128:18\n    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)\n    at invokeCallback (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:238:16)\n    at done (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:227:7)\n    at IncomingMessage.onEnd (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:287:7)\n    at IncomingMessage.emit (node:events:513:28)\n    at endReadableNT (node:internal/streams/readable:1359:12)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)","timestamp":"2025-03-17T17:33:13.319Z"}
{"level":"error","message":"SyntaxError: Expected double-quoted property name in JSON at position 58\n    at JSON.parse (<anonymous>)\n    at parse (D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\types\\json.js:92:19)\n    at D:\\projects\\recipes\\server\\node_modules\\body-parser\\lib\\read.js:128:18\n    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)\n    at invokeCallback (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:238:16)\n    at done (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:227:7)\n    at IncomingMessage.onEnd (D:\\projects\\recipes\\server\\node_modules\\raw-body\\index.js:287:7)\n    at IncomingMessage.emit (node:events:513:28)\n    at endReadableNT (node:internal/streams/readable:1359:12)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)","timestamp":"2025-03-17T17:33:47.297Z"}
{"level":"error","message":"Error login user:  secretOrPrivateKey must have a value","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (D:\\projects\\recipes\\server\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at D:\\projects\\recipes\\server\\src\\services\\auth.service.ts:43:20\n    at Generator.next (<anonymous>)\n    at fulfilled (D:\\projects\\recipes\\server\\src\\services\\auth.service.ts:5:58)","timestamp":"2025-03-17T17:34:22.994Z"}
{"level":"error","message":"No metadata for \"User\" was found.","timestamp":"2025-03-17T17:36:48.042Z"}
{"level":"error","timestamp":"2025-03-17T17:38:53.826Z"}
{"level":"error","timestamp":"2025-03-17T17:40:04.942Z"}
{"level":"error","timestamp":"2025-03-17T17:40:08.220Z"}
{"level":"error","timestamp":"2025-03-17T17:40:35.388Z"}
{"level":"error","timestamp":"2025-03-17T17:41:10.994Z"}
{"level":"error","timestamp":"2025-03-17T17:41:38.132Z"}
{"level":"error","timestamp":"2025-03-17T17:41:59.754Z"}
{"level":"error","timestamp":"2025-03-17T17:44:58.480Z"}
{"level":"error","message":"Error register user:  User with email exists","stack":"Error: User with email exists\n    at D:\\projects\\recipes\\server\\src\\services\\auth.service.ts:15:9\n    at Generator.next (<anonymous>)\n    at fulfilled (D:\\projects\\recipes\\server\\src\\services\\auth.service.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-17T17:54:27.426Z"}
